src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -889,14 +887,18 @@ private async Task<(long, object)> ModifyTimeSeriesConfiguration(TransactionOper
             var editTimeSeries = new EditTimeSeriesConfigurationCommand(configuration, name, raftRequestId);
             var result = await ServerStore.SendToLeaderAsync(editTimeSeries);
 
-            DatabaseTopology topology;
-            ClusterTopology clusterTopology;
+            List<string> members;
             using (context.OpenReadTransaction())
+                members = ServerStore.Cluster.ReadDatabaseTopology(context, name).Members;
+
+            try
+            {
+                await ServerStore.WaitForExecutionOnRelevantNodesAsync(context, members, result.Index);
+            }
+            catch (Exception e) when (e is AggregateException)
```

Comment:
Inside `WaitForExecutionOnRelevantNodesAsync`, we have the `HandleExceptions()` method, within which only two types of exceptions can be thrown:
- `TimeoutException`, in case `WaitForRaftIndexOnNodeAndReturnIfExceptionAsync` took too much time on all cluster nodes.
- `AggregateException`, which will wrap all errors of other types, and we want to catch this exception in the `WaitForExecutionOnRelevantNodesAsync` method so that we can form an `InvalidDataException` with a precise indication of the stage at which this error was thrown.

Reply:
will catch (RaftIndexWaitAggregateException e)